core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1021,6 +1021,20 @@ secret_configs:
       - cluster: app.ci
         namespace: ci
         name: build-farm-credentials
+  - from:
+      build01_vsphere_reg_auth_value.txt:
+        bw_item: build_farm
+        attachment: build01_vsphere_reg_auth_value.txt
+      build01_ci_reg_auth_value.txt:
+        bw_item: build_farm
+        attachment: build01_ci_reg_auth_value.txt
```

Comment:
shouldnt it be `build01_vsphere_reg_auth_value.txt`?

Reply:
The deployment job needs both

Reply:
ohoh.
of course.
My bad.